db/migrate/20221202083411_add_posts_ref_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddPostsRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments,
+                  :post, null: false, foreign_key: true
+  end
```

Comment:
- [ ] Also, kindly note that in your ERD diagram, there is supposed to be a author_id and not user_id as you have here. To resolve this, kindly refer to the comment above.

Reply:
This change has been done well